apps/client/src/layout/components/footer/footer.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import Link from "@cloudscape-design/components/link";
+import { FormattedMessage } from 'react-intl';
+import { colorBorderDropdownItemHover, colorBackgroundControlDefault, colorBackgroundLayoutToggleDefault, spaceScaledXs, spaceScaledL, spaceScaledXl } from '@cloudscape-design/design-tokens';
+import { footerColor, Links } from './constants'
+import './footer.css'
+
+export const Footer = () => {
+    
+    const [currentFooterColor, setCurrentFootercolor] = useState(footerColor);
+
+    const handleMouseEnter = (type:Links) => {
+        const hoverColor = {...footerColor, [type]:{color:colorBorderDropdownItemHover}}
+        setCurrentFootercolor(hoverColor);
+    };
+    const handleMouseLeave = (type:Links) => {
+        const defaultColor = {...footerColor, [type]:{color:colorBackgroundControlDefault}}
+        setCurrentFootercolor(defaultColor);
+    };
+    const footerStyle = {
+        color: colorBackgroundControlDefault,
+        backgroundColor: colorBackgroundLayoutToggleDefault,
+        padding: `${spaceScaledXs} ${spaceScaledL}`
+    };
+
+    const gapStyle = {
+        gap: spaceScaledXl
+    }
+    
+  return (
+    <footer id='app-footer' className="dashboard-footer-container" style={footerStyle} data-testid="footer-component">
+            <ul style={gapStyle}>
+                <li>
+                    <Link  data-testid="cloudshell-link" href="#">
```

Comment:
Sorry, i missed this before, can we have an array like 

`const someArray = [{testId: 'cloudshell-link', link: 'cloudshell', description: 'Open Cloudshell link'}, { feedback.... }]
`
and map thro in the html


```
<ul>
 { someArray.map(data =>  ( <li> ..... {data.description } </li>))}
</ui>
```

Reply:
resolved